build-system/tasks/extension-helpers.js

```diff
@@ -597,8 +609,23 @@ function buildExtensionCss(extDir, name, version, options) {
       })
     );
   }
-  promises.push(mainCssBinary);
-  return Promise.all(promises);
+
+  await Promise.all(parallel);
+}
+
+/**
+ * Build bento-*.css using the compiled amp-* result as source.
+ * It replaces all selectors for elements <amp-*> with <bento-*>.
+ * As a result of taking already minified code as source, this function is
+ * fairly fast and not cached.
+ * @param {string} name
+ * @param {string} minifiedAmpCss
+ * @return {!Promise}
+ */
+async function buildBentoCss(name, minifiedAmpCss) {
+  const bentoName = name.replace(/^amp-/, 'bento-');
+  const renamedCss = await renameSelectorsToBentoTagNames(minifiedAmpCss);
+  await fs.writeFile(`dist/v0/${bentoName}.css`, renamedCss);
```

Comment:
Other files output from this function go into `build/`, and later moved into `dist/`. 

I skipped `build/` altogether since the copying function expects an `amp-*` pattern. Happy to write into `build/` for consistency if desired, though we'd gain nothing more than consistency if I do so.